tests/core/test_tasks.py

```diff
@@ -476,6 +476,12 @@ def test_ts_generation_task(
     raw_study_path = tmp_path / "study"
 
     regular_user = User(id=99, name="regular")
+    jwt_user = Mock(
```

Comment:
But! it would be nice to add unit test to check that the transfer of the user to the task works correctly:
- submit a task to the task service that will just get the current user
- check that it indeed got the right user (the one that submitted the task)